.sparkswap/config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Sparkswap configuration for use inside the container. (Note the rpcCertPath is relative to being inside the container)
+ *
+ * It will use the user defined rpc address, user and password if they exist, otherwise
+ * they will use the default.
+ */
+
+module.exports = {
+  /**
+   * Address of the host for the Broker Daemon gRPC Server
+   * @type {String}
+   */
+  rpcAddress: process.env.RPC_ADDRESS || 'localhost:27492',
```

Comment:
`RPC_ADDRESS` will be something like `0.0.0.0:2749`, so should we just pull the port off here?